pos_sale_order_delivery/static/src/js/DeliveryNowOrLaterPopup.js

```diff
@@ -2,14 +2,14 @@
  * @author Raphaël Reverdy <raphael.reverdy@akretion.com>
  * License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). */
 
-odoo.define("pos_sale_order_delivery.DeliveryNowOrLaterPopUp", function (require) {
+odoo.define("pos_sale_order_delivery.DeliveryNowOrLaterPopup", function (require) {
     "use strict";
     /* global owl */
-    var AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
-    var {useState} = owl.hooks;
-    var Registries = require("point_of_sale.Registries");
+    const AbstractAwaitablePopup = require("point_of_sale.AbstractAwaitablePopup");
```

Comment:
why ?

Reply:
Isn't it a good practice to replace these declarations with let or const ?

Also, there is no need tu update it so this is probably why I chose const over let